ExternalModule.php

```diff
@@ -18,9 +19,38 @@ class ExternalModule extends AbstractExternalModule {
      * @inheritdoc
      */
     function redcap_data_entry_form_top($project_id, $record, $instrument, $event_id, $group_id) {
-        $this->includeJs('js/complexFieldValidationHelper.js');
+        $this->includeJs('js/checkRanges.js');
+
+        global $Proj;
+
+        foreach (array_keys($Proj->forms[$_GET['page']]['fields']) as $field_name) {
+            $field_info = $Proj->metadata[$field_name];
+            
+            if (!$display_mode = Form::getValueInActionTag($field_info['misc'], $name)) {
```

Comment:
display_mode is an odd variable name.  How about listOfValuesAssignedToActionTag.

$name is not set. Also, a better variable name would be $actionTagName